lib/pages/dashboard/dashoard_page.dart

```diff
@@ -140,12 +138,17 @@ class _DashboardPageState extends State<DashboardPage> {
                 ],
               )
             : Container(
-                constraints: BoxConstraints(minHeight: 0),
+                constraints: BoxConstraints(minHeight: 100),
+                child: Center(child: Loading()),
               );
       case 2:
         return Paragraph(title: "Vos équipes");
       case 3:
-        return DashboardClubTeams(club: _currentClub, onTeamFavoriteChange: (_) => _selectCurrentClub(_currentClub));
+        return DashboardClubTeams(
+          club: _currentClub,
+          onTeamFavoriteChange: (_) => _selectCurrentClub(_currentClub),
```

Comment:
Don't you have to reload the calendar in this case? a team may have been removed from the favorite!